### ISSUE TITLE
Human Fragility

### ISSUE THREAD

**Issue by Mr-Satan-Official** [studio-minus/ppg-bugs-and-requests#460]
**_Make Humans Less Fragile_**
So I'm not sure if it's correct to put this under features, since it's more of a complaint, but regardless, I have to say that I'm not a huge fan of how fragile humans have become in 1.22. In particular, I dislike the new head trauma system. It's an interesting concept, but in practice, if you enjoy acting out little scenarios with the humans like I do, it just makes it incredibly frustrating when the collapse from the smallest bump. To add to this, the mod I had that could be used to wake entities from unconsciousness no longer works with this version, which admittedly is an issue for the mod developer to sort out, but I still thought I'd mention it as a gripe. So in short, I think making humans less fragile and reducing the head trauma effect would be a good idea.

_**Alternatively, add "Classic Humans"**_
A solution that might work for both fans of the old humans and fans of the new ones, is adding either a separate entity that function like the old humans, or an optional setting for the new mechanics that can be enabled or disabled at will. That way people who enjoy either version can continue to enjoy it.

**Comment by Mr-Satan-Official** [comment]
Also apologies if this has been brought up already, I did a quick scan of other threads to see if the issue had been raised but not a thorough search

**Comment by bluewoods64** [comment]
true, humans are just so weak now and i can barely get them in a position without them getting brain damage.
